src/AppInitializer.tsx

```diff
@@ -110,6 +114,10 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
           {React.Children.map(this.props.children, (child: any) => {
             return child
           })}
+          {/* Force synapse.org login state to be in sync with this app. 
+          Note that this is only rendered after hasCalledGetSession is set to true. */}
+          {this.state.token && <HiddenIFrame url={`https://signin.synapse.org/login?code=${this.state.token}`} />}
```

Comment:
Could you explain what this is doing? From my understanding, when the user has a token it renders the HiddenIFrame which calls the signin for Synapse then signing in the user. Please correct me if I am wrong.